recipes/flatbuffers/all/conanfile.py

```diff
@@ -82,7 +82,8 @@ def _configure_cmake(self):
         self._cmake.definitions["FLATBUFFERS_BUILD_FLATLIB"] = self.options.flatbuffers and not self.options.shared
         self._cmake.definitions["FLATBUFFERS_BUILD_FLATC"] = self.options.flatc
         self._cmake.definitions["FLATBUFFERS_BUILD_FLATHASH"] = False
-        self._cmake.definitions["FLATBUFFERS_STATIC_FLATC"] = not self.options.shared
+        if self.settings.os != "Macos":
+             self._cmake.definitions["FLATBUFFERS_STATIC_FLATC"] = not self.options.shared
```

Comment:
```suggestion
        self._cmake.definitions["FLATBUFFERS_STATIC_FLATC"] = False
```
let's just set it to the false always (see comments)